test/run-openshift-remote-cluster

```diff
@@ -23,6 +23,8 @@ ct_os_set_ocp4
 
 ct_os_check_compulsory_vars
 
+ct_os_tag_image_for_cvp "redis"
```

Comment:
Is there a chance, that this would be ran for non-rhel container?
I see that the image can be tagged either `-el7` or `-el8`. What if we test for example centos images?